doc/api/net.md

```diff
@@ -822,6 +822,10 @@ behavior.
 <!-- YAML
 added: v0.1.90
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/41310
+    description: The `noDelay`, `keepAlive` and `keepAliveInitialDelay`
+                 options are supported now.
```

Comment:
This should have gone to its own PR, since this PR is semver-major it will land on v18.0.0, while https://github.com/nodejs/node/pull/41310 will land on v17.7.0.